tool/site/client/src/components/dose-to-text/doseToTextResult.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react"
+import {CrossIcon, Label, Table, TickIcon} from "nhsuk-react-components"
+import ButtonList from "../common/buttonList"
+import BackButton from "../common/backButton"
+import LongRunningTask from "../common/longRunningTask"
+import MessageExpanders from "../messageExpanders"
+import {ApiResult} from "../../requests/apiResult"
+import {DosageTranslation} from "../../../../server/src/routes/dose-to-text"
+
+interface DoseToTextResultProps {
+  task: () => Promise<DoseToTextApiResult>
+}
+
+export interface DoseToTextApiResult extends ApiResult {
+  results: Array<DosageTranslation>
+}
+
+const DoseToTextResult: React.FC<DoseToTextResultProps> = ({
+  task
+}) => {
+  return (
+    <LongRunningTask<DoseToTextApiResult> task={task} loadingMessage="Translating structured dosage to text.">
+      {doseToTextResult => (
+        <>
+          <Label isPageHeading> Dose to Text Result {doseToTextResult.success ? <TickIcon /> : <CrossIcon />}</Label>
```

Comment:
Not for this PR but a small component that hands the `success ? <TickIcon /> : <CrossIcon />` logic that we have everywhere on results pages might be nice

Reply:
Agreed but not for this PR